### ISSUE TITLE
Window switcher hotkey some not update properly

### ISSUE THREAD

**Issue by RubikOwl** [ejbills/DockDoor#130]
Changing the hotkey for the window switcher does not update, and does not always recognize custom hotkeys. I wanted to use ⌥ + ⇥ but had to do ⌥ + ⇥ + shift, at which point it switched to ⌥ + ⇥.

While I'm here, a couple notes. The UI for the window switcher is very nice! A couple things I'd like to see tweaked would be having the windows be in order of use, not alphabetical. Staying open when closing an app from in the switcher would be useful, and having multiple layers when too many windows are open would be nice as well. Loving the app so far, hoping to fully replace a couple different ones I use!

**Comment by hasansultan92** [comment]
Hi @RubikOwl, can you elaborate on the first issue you have mentioned a little bit more. I am working with a v1.1.2 application and I am having no trouble switching to option + tab as the keybind of choice. The default if not cmd + tab is control + tab. You are not able to have three keys enable the window switcher the you have mentioned with option + tab + shift.

**Comment by hasansultan92** [comment]
With regards to the second enhancement, thats actively being worked on, see #196, see #116 ,so that would be a duplicate

**Comment by RubikOwl** [comment]
Hi. I'm on version 1.1.4 now. I'm actually still having issues with changing the switcher in settings, though it has stayed on option-tab since my roundabout way of fixing it. Clicking on the three options available do not do anything, though enabling the "use default MacOS" option does toggle between command-tab and the keybind that I set. Not sure why I'm having this issue and not you, though it may be due to other apps I use.

One other thing I found as I've been using the app switcher is that it has a bit on a delay before it pops up, maybe like a second. It doesn't do that though when the settings pane for dockdoor is open, it's quite responsive then.
